net/dns/resolved.go

```diff
@@ -42,6 +42,22 @@ var resolvedListenAddr = netaddr.IPv4(127, 0, 0, 53)
 
 var errNotReady = errors.New("interface not ready")
 
+// DBus entities we talk to.
+//
+// DBus is an RPC bus. In particular, the bus we're talking to is the
+// system-wide bus (there is also a per-user session bus for
+// user-specific applications).
+//
+// Daemons connect to the bus
+const (
+	dbusResolvedObject    = "org.freedesktop.resolve1"
+	dbusResolvedPath      = "/org/freedesktop/resolve1"
```

Comment:
want to make these ObjectPath ones be typed constants?

https://pkg.go.dev/github.com/godbus/dbus/v5#ObjectPath

(didn't look at whether there are types for the others)